test/acceptance/registry-ui.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const expect = require('chai').expect;
+const oc = require('../../src/index');
+const path = require('path');
+const request = require('minimal-request');
+
+describe('registry (ui interface)', () => {
+
+  let registry, result, error, headers;
+
+  const next = (done) => (e, r, d) => {
+    error = e;
+    result = r;
+    headers = d.response.headers;
+    done();
+  };
+
+  const conf = {
+    local: true,
+    path: path.resolve('test/fixtures/components'),
+    port: 3030,
+    baseUrl: 'http://localhost:3030/',
```

Comment:
I would be able to, not sure why should I?

The purpose of this tests is mostly ensure that we have a kind of smoke test for when we change the registry so that the UI doesn't break. Tests for testing the "prefix" parameter (what would be like a `/v2/` segment - are already there on a unit level.